doc/stdlib/parallel.rst

```diff
@@ -173,29 +173,19 @@ General Parallel Computing Support
 
    Keyword arguments:
 
-   * ``tunnel``\ : if ``true`` then SSH tunneling will be used to connect to the worker from the             master process. Default is ``false``\ .
-
+   * ``tunnel``\ : if ``true`` then SSH tunneling will be used to connect to the worker from the           master process. Default is ``false``\ .
    * ``sshflags``\ : specifies additional ssh options, e.g.
 
-   .. code-block:: julia
-
-       sshflags=`-i /home/foo/bar.pem`
-
-   * ``max_parallel``\ : specifies the maximum number of workers connected to in parallel at a host.                   Defaults to 10.
-
-   * ``dir``\ : specifies the working directory on the workers. Defaults to the host's current          directory (as found by ``pwd()``\ )
-
-   * ``exename``\ : name of the ``julia`` executable. Defaults to ``"$JULIA_HOME/julia"`` or              ``"$JULIA_HOME/julia-debug"`` as the case may be.
-
+     sshflags=``-i /home/foo/bar.pem``
```

Comment:
Same problem as with `IOContext`, using a fenced code block should fix this too.